### PR TITLE
Api analytics

### LINKED ISSUE #22: Recipe: API analytics
Perform API usage logging without interfering with the UI layer by adding a service worker to gather the usage and use the `sync` API to upload gathered data from time to time.

### PR DESCRIPTION
**Pull request by delapuente**
Fixes #22 
@zalun @darkwing  , do you want to take a look?

### CODE REVIEW

File: api-analytics/index.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
+var ENDPOINT = 'api/quotations';
+
+// Register the worker and show the list of quotations.
+navigator.serviceWorker.register('worker.js').then(function() {
+  loadQuotations();
+});
+
+// When clicking add button, get the new quote and author and post to
+// the backend.
+document.getElementById('add').onclick = function() {
```
Comment (darkwing) [inline api-analytics/index.js]: Sort of a nit but can we use a real `<form>` and do `onsubmit` so users can add a new quote upon `enter` key?

Comment (delapuente) [inline api-analytics/index.js]: Sure, nice idea.

**Comment by delapuente** [comment]
@darkwing do you want to take a second look here? I think it's ready to merge.

**Comment by darkwing** [comment]
Works perfectly, great job @delapuente !  R+

**Pull request merged**